File: source/clibCore/Models/Context/stylesheetContextModel.vb

Option Explicit On
Option Strict On

Imports Contensive.Core.Controllers
Imports Contensive.Core.Controllers.genericController

Namespace Contensive.Core.Models.Context
    '
    '====================================================================================================
    ''' <summary>
    ''' Site Properties
    ''' </summary>
    Public Class stylesheetContextModel
        Public Property templateId As Integer
        Public Property EmailID As Integer
        Public Property StyleSheet As String
    End Class
End Namespace

File: source/core/Models/Context/stylesheetContextModel.cs

using System;
using System.Reflection;
using System.Xml;
using System.Diagnostics;
using System.Linq;
using System.Text.RegularExpressions;
using System.Collections.Generic;
using System.Data;
using System.Data.SqlClient;
using Contensive.Processor;
using Contensive.Processor.Models.DbModels;
using Contensive.Processor.Controllers;
using static Contensive.Processor.Controllers.genericController;
using static Contensive.Processor.constants;
//
namespace Contensive.Processor.Models.Context {
    //
    //====================================================================================================
    public class stylesheetContextModel {
        public int templateId { get; set; }
        public int EmailID { get; set; }
        public string StyleSheet { get; set; }
    }
}